packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -13,8 +13,9 @@ const UserValidationMessage = {
 	EMAIL_REQUIRE: "Email is required",
 	EMAIL_TAKEN: "Email is taken",
 	EMAIL_WRONG: "Please enter a valid email address",
+	FIELD_REQUIRED: "This field is required",
 	NAME_INVALID_CHARACTERS:
-		"Please enter a valid name. Only letters, numbers, hyphens, and underscores are allowed",
+		"Please enter a valid name. Your name must be between 3 and 40 characters. Only Latin letters, numbers, hyphens, and underscores are allowed",
```

Comment:
WIll this fit under input field?

Reply:
It's gonna be a few rows. Requirements document demands to use this message here.

Reply:
Yeah, for the all requirements you can check in the description